Rakefile

```diff
@@ -263,7 +263,9 @@ namespace :package do
     s3_path = is_standalone ? Pathname.new(ZIPPED_STANDALONE) : Pathname.new(ZIPPED_BUNDLE)
     latest_obj = s3_bucket.objects[latest_path].write(s3_path)
     fastlane_obj = s3_bucket.objects[fastlane_path].write(s3_path)
+    latest_obj.content_disposition = 'attachement; filename="fastlane.zip'
```

Comment:
Peeping at the docs for http://docs.aws.amazon.com/AWSRubySDK/latest/AWS/S3/S3Object.html, I don't think it has a method `content_disposition`.

You either need to use the `ObjectMetadata`'s ability to update the metadata or `copy_from` since `[]` is deprecated on `ObjectMetadata`: http://docs.aws.amazon.com/AWSRubySDK/latest/AWS/S3/S3Object.html#copy_from-instance_method

http://docs.aws.amazon.com/AWSRubySDK/latest/AWS/S3/ObjectMetadata.html#%5B%5D%3D-instance_method